### PR TITLE
chore(repo): Remove Todo

### PR DESCRIPTION
**Pull request by MaxBittker**
guess we did it all

### CODE REVIEW

File: TODO [hunk 1]
```diff
@@ -1,8 +0,0 @@
-- API needs to work with key authentication globally (not just the store view)
-- Record errors with client information in API
-- Record client information with each event
-- Pause/Play, Sort By, and Notification settings should be saved in a cookie
-- Potentially add graphs per server/url/etc ?
```
Comment (mattrobenolt) [inline TODO]: cc @tkaemming @bretthoerner :)

**Review by mattrobenolt — APPROVED** [review]
@MaxBittker to be fair, we've barely done anything from this. But I also agree that a TODO is not exactly how we track how we develop Sentry in 2017. 🍪

File: TODO [hunk 2]
```diff
@@ -1,8 +0,0 @@
...
-- Make uninteresting frames in stacktraces opaque
-- Create tailer for apache-style error logs
```
Comment (JTCunning) [inline TODO]: lol

File: TODO (hunk 1 above)
Comment (tkaemming) [reply TODO]: 🤔

**Pull request merged**